File: Java Labs/Patterns On Java/src/com/pattern/proxy/RealClothingImage.java
package com.pattern.proxy;

public class RealClothingImage implements ClothingImage {
    private String imageUrl;

    public RealClothingImage(String imageUrl) {
        this.imageUrl = imageUrl;
        loadImageFromDisk();
    }

    private void loadImageFromDisk() {
        System.out.println("Loading image from disk: " + imageUrl);
    }

    @Override
    public void display() {
        System.out.println("Displaying image: " + imageUrl);
    }
}

File: Python Labs/com/pattern/proxy/RealClothingImage.py
from com.pattern.proxy.ClothingImage import ClothingImage


class RealClothingImage(ClothingImage):
    def __init__(self, image_url):
        self.image_url = image_url
        self.load_image_from_disk()

    def load_image_from_disk(self):
        print(f"Loading image from disk: {self.image_url}")

    def display(self):
        print(f"Displaying image: {self.image_url}")